test/integration/consul-container/libs/node/consul-container.go

```diff
@@ -112,25 +117,37 @@ func NewConsulContainer(ctx context.Context, config Config) (Node, error) {
 
 	return &consulContainerNode{
 		config:    config,
-		container: container,
+		pod:       podContainer,
+		container: consulContainer,
 		ip:        ip,
 		port:      mappedPort.Int(),
 		client:    apiClient,
 		ctx:       ctx,
-		req:       req,
+		podReq:    podReq,
+		consulReq: consulReq,
 		dataDir:   tmpDirData,
 	}, nil
 }
 
-func newContainerRequest(config Config, name, configFile, dataDir, license string) testcontainers.ContainerRequest {
+const pauseImage = "k8s.gcr.io/pause:3.3"
```

Comment:
i'm a bit biased to put const at the top or bottom of the file for readibility; do you have any preferences?